src/pages/index.tsx

```diff
@@ -1,12 +1,124 @@
+          existing verification tools.
+        </>
+      ),
+    },
+    {
+      question: 'How does it work?',
+      answer: (
+        <>
+          Instead of requiring you to submit a large JSON file or flattened solidity code, Cove asks
+          for git information and framework settings (e.g. use forge with the{' '}
+          <code className='text-sm'>default</code> profile) for the contracts you want to verify.
+          For now it also asks for transaction hashes, but we plan to remove this in the future.
+        </>
+      ),
+    },
+    {
+      question: 'What are the limitations?',
+      answer: (
+        <>
+          Cove is in alpha, so it currently has a few limitations that will be lifted in the future:
+          <ul className='list-disc pl-8'>
+            <li>Only forge projects are supported.</li>
+            <li>Only Solidity contracts are supported.</li>
+            <li>
+              Verified contracts are not saved in a database anywhere. Verifying contracts on the{' '}
+              <Link href='/verify' className='hyperlink'>
+                verify
+              </Link>{' '}
+              page solely helps us improve Cove before the official release (and hopefully gets you
+              excited about the future of contract verification).
+            </li>
+            <li>
+              To reduce our expenses for now, we do not yet trace calls or search for deployment
+              transactions. This means:
+              <ul className='list-disc pl-8'>
+                <li>You must provide the transaction on the verification form.</li>
+                <li>
+                  Contract deployed from arbitrary contracts cannot yet be verified. Only contracts
+                  deployed via a standard <code className='text-sm'>create</code> deployment, Nick
+                  Johnson&apos;s <code className='text-sm'>create2</code>{' '}
+                  <ExternalLink
+                    text='deployer'
+                    href='https://github.com/Arachnid/deterministic-deployment-proxy'
+                  />
+                  , or 0age&apos;s <code className='text-sm'>create2</code>{' '}
+                  <ExternalLink
+                    text='deployer'
+                    href='https://etherscan.io/address/0x0000000000FFe8B47B3e2130213B802212439497'
+                  />{' '}
+                  are supported.
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </>
+      ),
+    },
+    {
+      question: 'I like Cove, how can I help it succeed?',
+      answer: (
+        <>
+          Try{' '}
+          <Link href='/verify' className='hyperlink'>
+            verifying
+          </Link>{' '}
+          a contract! This is a public goods project funded solely by hackathon{' '}
+          <ExternalLink text='prizes' href='https://ethglobal.com/showcase/cove-8ohfk' />, so
+          consider donating to TODO or nominating us for the{' '}
```

Comment:
Should we remove this for now, since there's not yet a place to donate and optimism retropgf is not open?

Reply:
Added donation address in ab3ad75